src/types.ts

```diff
@@ -15,11 +15,11 @@ export interface IGetArguments<Properties> {
   properties?: (keyof Properties)[];
 }
 
-export interface ISetArguments<CreatedObject> {
+export interface ISetArguments<Foo> {
   accountId: string;
-  ifInState?: string | null;
-  create?: { [temporaryId: string]: CreatedObject };
-  update?: { [id: string]: { [jsonPointer: string]: any } };
+  ifInState?: string;
+  create?: { [id: string]: Partial<Foo> };
+  update?: Partial<Foo>;
```

Comment:
```suggestion
  ifInState?: string | null;
  create?: { [temporaryId: string]: Partial<Foo> };
  update?: Partial<Foo> & { [id: string]: { [jsonPointer: string]: any } };
```

Reply:
I did I mistake, the Partial<Foo> should be just before the `{ [jsonPointer: string]: any }`